.gitpod.yml

```diff
@@ -31,7 +31,7 @@ vscode:
     - golang.go
     - stylelint.vscode-stylelint
     - DavidAnson.vscode-markdownlint
-    - johnsoncodehk.volar
+    - vue.volar
```

Comment:
```suggestion
    - Vue.volar
```

https://marketplace.visualstudio.com/items?itemName=Vue.volar

<img width="307" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/c9d414ee-70ad-4ceb-873a-559e6a8de747">

Reply:
Yeah those are likely case-insensitive anyways, but I trust the vscode store more.

Reply:
Just FYI Gitpod doesn't use that marketplace. https://www.gitpod.io/docs/references/ides-and-editors/vscode-extensions